packages/core/src/components/region/RegionPopover/RegionPopover.tsx

```diff
@@ -157,6 +157,11 @@ function RegionPopover({
             offsetTop={offsetTop}
             offsetLeft={offsetLeft}
             closeButtonAriaLabel={closeButtonAriaLabel}
+            onEntered={() => {
+              if (!postalCode && inputRef.current) {
```

Comment:
question: when there is a postalCode and the popover is open (postalCode default), where should be the focus?

Reply:
it will be the default behaviour, no element will be focused.